sdk/formrecognizer/Azure.AI.FormRecognizer/MigrationGuide.md

```diff
@@ -404,7 +404,7 @@ foreach (DocumentPage page in result.Pages)
 
         Console.WriteLine($"    Its bounding polygon (points ordered clockwise):");
 
-        for (int j = 0; j < line.BoundingPolygon.Points.Length; j++)
+        for (int j = 0; j < 4; j++)
```

Comment:
All bounding polygons have a fixed length of 4. The service team wants to increase the number of points to better fit the visual elements in the future. Hardcoding to 4 works until we expose a `Length` property in the next release.